package.json

```diff
@@ -10,7 +10,7 @@
   },
   "main": "src/main.js",
   "scripts": {
-    "start": "goat -e index.html -p 8000",
+    "start": "goat -e dist/index.html -p 8000",
```

Comment:
currently `grunt` jumps into a infite loop when I try to `serve` or `watch`. It's probably just my dev environment, I'll have to figure it out more. But until then let's keep this around as an option.